### ISSUE TITLE
Download Status Window

### ISSUE THREAD

**Issue by blackcrack** [erickutcher/httpdownloader#55]
Hi,

would be nice if exist a "download Status Window" for any File where be downloaded
to see the download time, the %Bar, and other Information's to have a direct Overview 
to each single file where is downloaded.. 
Maybe Skin-able with Variables and x/y of the information's-Window
in a ini-file in own Skin-directory.. maybe something like Winamp-skins(in zip files, readable with infozip dll) or in open skins/sub-directorys .

to have a possible for Adaptation via skins to the Users System-design :) 
(also the modded Surfaces of Windows)

best regards
Blacky

**Comment by erickutcher** [comment]
I think it would be easier to just look at the main window. It contains all the information you'd need. You could enable the progress bar for the status bar icon and then just hover your mouse over it for some extra info. Skinning is not something that I like in programs. It tends to bloat them up and they always get broken with updates. Personally, I like a simple utilitarian design that matches the style of the programs that are native to an operating system.

**Comment by blackcrack** [comment]
naa, some peoples want have it visual with any each single advance/prestep window for any download,
You must it not do, you could easy give the possibility , a dll for let see a Picture and with the X/Y in a ini where be the information's pinned and where it is information and the progress bar.
some peoples don't like to have ever the head Window open and want see the single downloaqded files and disappears if they are finish, this is the idea behind.. and not have ever the big head window in the front to know to be this or the other file in progress, therefor for one file, one Status progress window.. and disappear if finish, you see ?
this could be possible with a 

[X] let see the Download Progress in a single window
[X] select a Skin [_./skin/slanXP2_____[^] // where select a skin-folder in the program folder

this can use a skin.ini with the background-picture & the X/Y variables of the %informations% variables 
Downloadtime %dtime%, NetworkSpeed %nspeed% and so on in the inifile
and Progressbar (%pbar%), 
pbackgroundcolor %pbarbcolor%, %pbarprocol1%,%pbarprocol50% , %pbarprocol100% or %pbarprocolspin1%,%pbarprocolspin2% ...

the users can make the skins... like at the xmplay .. this is also a "simple" player but gives the Peoples to customize the player at hand .. and it is the fully success .. and the result is the big popularity , why, the peoples can customize the player, like they want.. 

Also the Freedownloadmanager 1/2 had a very bug sugsess, why ? Alex give the single download window... for each file .. i have give him the suggestion .. you can ask him.. since the FDM no more it is like it is and the single downloadwindow is disappear in vers.5 don't like it him.. but other peoples have  set through :\ now do i search a lightweight downloadmanager with single download-window for any file where disappear at download end .. and it can be your down loader where full fill this this gap ..
if you support a download-window .. and you can be sure that it will be a success and more people will come when it is programmed in ..
you could also create it as snapin ... like how in xmplay the plugins.. as example
therewith can you hold your head programm fast as possible and give the peoples the possible for make enhancement/extensions for the Program  ..

best regards
Blacky

**Comment by Zero3K** [comment]
I also agree with the suggestion of having a window show up for each download like how Internet Download Manager, etc. does it.

**Comment by smaragdus** [comment]
> I also agree with the suggestion of having a window show up for each download like how Internet Download Manager, etc. does it.

Old-style download managers usually have options to show separate windows for separate downloads. Modern download managers like Free Download Manager (since version 5), EagleGet, Xtreme Download Manager, etc, have eliminated these windows. Although I abhor these modern download managers (I regard all of them as dumbed-down abominations for dumbed-down people) I regard the removal these separate windows as a good idea- I find such windows distracting and useless. For me HTTP Downloader offers enough progress indicators for downloads.

**Comment by Zero3K** [comment]
I think that the separate download windows should be an option to use/not use if it is implemented.
